accepted/2020/workloads/workload-manifest.md

```diff
@@ -0,0 +1,430 @@
+# .NET SDK Workload Manifests
+
+# Overview
+
+.NET SDK workload manifests describe workloads that are available to be installed in the .NET SDK.  They provide version mappings that allow MSBuild targets and other SDK tools to resolve the packs that comprise these workloads from the SDK installation directory, and can be used to acquire, repair, and update the workloads and their components.
+
+This document describes the format and versioning of the manifest. The installation and update experience will be described in other documents.
+
+# Definitions
+## SDK Band
+
+.NET SDK versions take the form 3.1.100, 3.1.101, 3.1.200. The first two components of this version represent the major and minor version of the runtime in the SDK. The third component (the “hundreds”) is a more complex: the first digit represents the feature release of the SDK, and the last two digits represent the patch level of the SDK and runtime combined.
+
+In the context of this document we will refer to an *SDK band* as the common portion of the version of all the SDK releases that differ only by patch level. For example, 3.1.100 and 3.1.105 are in the 3.1.100 band, while 3.2.100 is in the 3.2.100 band and 3.1.203 is in the 3.1.200 band.
+
+## SDK Pack
+
+An SDK pack is a unit of packaging for the assets that make up the SDK. Packs have an ID and version and are installed into folders in the SDK directory.
+
+Packs will be installed using CLI tooling or by a native installer such as the Visual Studio installer. It is likely that packs will be distributed via NuGet at some point, so it is recommended that pack authors reserve NuGet IDs for their packs even if other distribution methods are used initially.
```

Comment:
> it is recommended that pack authors reserve NuGet IDs

Makes sense in the broad. Can't the SDK team just reserve the right top-level names, and call it good? We should be like a TLD owner.

Reply:
Probably? I assume most packs would fall under `System.*` or `Microsoft.*` which are already reserved prefixes. Added a note of clarification.